catalog/ui/catalog-ui-search/webpack/config/dev.js

```diff
@@ -8,6 +8,16 @@ var resolve = function (place) {
     return path.resolve(__dirname, '../../', place)
 };
 
+var handleProxyRes = function (proxyRes, req, res) {
+    // remove so we can still login in through http
+    delete proxyRes.headers['x-xss-protection'];
+    var cookie = proxyRes.headers['set-cookie'];
+    if (cookie !== undefined) {
+        // force the cookie to be insecure since the proxy is over http
+        proxyRes.headers['set-cookie'] = cookie[0].replace(/;Secure/, '')
```

Comment:
This code has worked with firefox in the past. I believe the space is just stylistic.